src/terra-dev-site/doc/application/how-to.2/manage-pending-actions.doc.mdx

```diff
@@ -27,9 +27,11 @@ const ExampleComponent = () => {
 export default ExampleComponent;
 ```
 
-[ApplicationBase](/components/terra-application/application/reference/components/application-base) will check for the presence of NavigationPrompts rendered within it before it allows the window to be unloaded. Please see its reference page for more information.
+Within `terra-application`, the following components respond to the presence of NavigationPrompts:
 
-[ModalManager](/components/terra-application/application/reference/components/modal-manager) and [SlidePanelManager](/components/terra-application/application/reference/components/slide-panel-manager) will check for the presence of NavigationPrompts within their disclosed components and prompt the user if necessary before allowing the components to be dismissed. Please see their respective reference pages for more information.
+* [ApplicationBase](/components/terra-application/application/reference/components/application-base) - NavigationPrompts rendered within children will result in prompting prior to window unload events.
+* [ModalManager](/components/terra-application/application/reference/components/modal-manager)/[SlidePanelManager](/components/terra-application/application/reference/components/slide-panel-manager) - NavigationPrompts rendered within disclosed components will result in prompting prior to disclosure dismissal.
```

Comment:
**If** NavigationPrompts **are** rendered within disclosed components, **the user** will ~result in prompting~ **be prompted before the content can be dismissed** ~prior to disclosure dismissal.~